sdk/keyvault/azkeys/CHANGELOG.md

```diff
@@ -1,10 +1,16 @@
 # Release History
 
-## 0.2.1 (Unreleased)
+## 0.3.0 (Unreleased)
 
 ### Features Added
+* Adds the `CreateOKPKey` method for creating OKP public keys
+* Adds the `Ed25519` `KeyCurveName`
+* Adds the `OKP` and `OKPHSM` `KeyType`s
 
 ### Breaking Changes
+* `NewClient` returns an instance of a `Client`, instead of a `*Client`
```

Comment:
What's motivating this? I thought we'd aligned on returning pointers, which is still [the guideline](https://azure.github.io/azure-sdk/golang_introduction.html#service-client-constructors).

Reply:
Correct, we should be returning clients by reference.